product_docs/docs/pgd/4/rel_notes/pgd_4.2.1_rel_notes.mdx

```diff
@@ -0,0 +1,20 @@
+| BDR       | 4.2.1   | Bug fix | Grant bdr.replication_status to bdr_read_all_stats (BDR-2523) <p>  Alllow the read_all_stats role to check BDR replication information. </p>                                                                                                                                                                             |
+| BDR       | 4.2.1   | Bug fix | Resolve hang related to `bdr.standby_slots_min_confirmed` (BDR-2445, RT80692) <p> Allow the WAl sender to proceed without the need to wait for the timeout after a Subscriber Only node is parted, while using standby_slot_names. </p>                                                                                  |
+| BDR       | 4.2.1   | Bug fix | Use two_phase slots in `bdr_init_physical` on PG14+ () <p> Improve prepared transaction handling during `bdr_init_physical`. </p>                                                                                                                                                                                        |
+| BDR       | 4.2.1   | Bug fix | Add schema name to relation name in on all error messages (BDR-765) <p> Improve table information in all error messages. </p>                                                                                                                                                                                            |
+| BDR       | 4.2.1   | Bug fix | Allow parting a node only if the node didn't already started the part process (BDR-2629) <p> Return with error if a node is already in at least PART_START state. </p>                                                                                                                                                   |
+| BDR       | 4.2.1   | Bug fix | Don't wait for ADD CONSTRAINT progress if DDL replication is off (BDR-2645, RT86043) <p> The constraint validation from all nodes is not needed if we don't replicate the DDL or from any node that is PARTED or STANDBY. </p>                                                                                           |
+| BDR       | 4.2.1   | Bug fix | Adjust `replication_set_add_ddl_filter` (BDR-2755, RT86825) <p> Replicate all the columns needed for the ddl_filter. </p>                                                                                                                                                                                                |
+| BDR       | 4.2.1   | Bug fix | Add maintanance to bdr.queue catalog table (BDR-2475, RT84105) <p> Remove automatically rows older than the `bdr.max_queue_days` (default disabled) on bdr.queue. </p>                                                                                                                                                   |
```

Comment:
s/Remove automatically/Automatically remove/

I would be more explicit with the default and add a separate sentence: "By default, entries are retained forever to match the existing behaviour."